html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+
+</script>
+</head>
+<body>
+<script>
+setup({explicit_done : true});
+
+function testTextDecorationSkip() {
+      test(
+          function() {
+              var computed_text_decoration_skip =
+                  getComputedStyle(
+                      document.body)["text-decoration-skip"];
+              assert_regexp_match(
+                  computed_text_decoration_skip,
```

Comment:
Does the spec not define the order? Please file a bug on the spec if so, and link to it here.

Reply:
Not applicable anymore since the new property is not multi-valued.